.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
   build-clang:
     runs-on: ubuntu-latest
     env:
-      CC: clang
+      CC: clang-15
```

Comment:
We can make the modification to the code and add 'void' but I am not sure it is good idea to bind specific varsion of clang. There are pros and cons of selecting new/old clang (unless we choose both and taking care they are available on the machine and install it if not) from that point IMHO we better leave whatever is on the machine.

Reply:
ok, then we should remember this issue (clion already highlights it in IDE).